Makefile

```diff
@@ -39,7 +39,7 @@ endif
 
 .PHONY: format
 format: $(INSTALL_STAMP) ## Apply formatters
-	$(POETRY) run isort --profile=black --lines-after-imports=2 .
```

Comment:
Did you mean to change this?

Reply:
Yes this was intentional, since we now have the `isort` config in `pyproject.toml`